### ISSUE TITLE
Colonisation, Wastelands and Scripted Functions

### ISSUE THREAD

**Issue by Science-Recon** [PhelanBavaria/ancienttimeline#61]
While it might not be a priority, a way to remove the wastelands from the map without breaking the game and the balance (And could possibly be added to ET, too) would be to use the scripted function `can_colonize_province = { }` and use a province modifier and then specify in the scripted function that you cannot colonise unless you have a certain tech/culture/religion or some other requirement. It could also have a attrition and movement speed penalty or something like that.

In addition, since Russia isn't a thing in AT, we can use Siberian frontiers (and the other Russian mechanics) as well.

Thoughts?

**Comment by qweytr** [comment]
That's a good idea for ET, but I'm not sure how well it would fit here.

And I would like to have all the vanilla mechanics used somewhere in this mod, but I don't think there were any countries during the AT timeline that did large scale colonization by land. Almost all the colonization was done by sea by the Greeks and the Phoenicians.

**Comment by Firesoul7** [comment]
The Illyrians too. I'd have to look into the matter of land based colonisation, but my impression is that it did happen, it was just slower because with a few exceptions (like Amarna) it tended to be a fairly organic process, with village splitting off from village and farming then settling new lands.
-------- Original message --------From: qweytr <notifications@github.com> Date: 18/10/2017  08:06  (GMT+00:00) To: PhelanBavaria/ancienttimeline <ancienttimeline@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: Re: [PhelanBavaria/ancienttimeline] Colonisation, Wastelands and Scripted Functions (#61) 
That's a good idea for ET, but I'm not sure how well it would fit here.
And I would like to have all the vanilla mechanics used somewhere in this mod, but I don't think there were any countries during the AT timeline that did large scale colonization by land. Almost all the colonization was done by sea by the Greeks and the Phoenicians.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/PhelanBavaria/ancienttimeline","title":"PhelanBavaria/ancienttimeline","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/PhelanBavaria/ancienttimeline"}},"updates":{"snippets":[{"icon":"PERSON","message":"@qweytr in #61: That's a good idea for ET, but I'm not sure how well it would fit here.\r\n\r\nAnd I would like to have all the vanilla mechanics used somewhere in this mod, but I don't think there were any countries during the AT timeline that did large scale colonization by land. Almost all the colonization was done by sea by the Greeks and the Phoenicians."}],"action":{"name":"View Issue","url":"https://github.com/PhelanBavaria/ancienttimeline/issues/61#issuecomment-337479835"}}}

**Comment by Science-Recon** [comment]
Yeah, most of the colonisation was done in the Mediterranean, by the Greeks and Phoenicians. This as well is a bit problematic, because most of their colonisation was on already inhabited lands and EUIV doesn't really support that. Maybe a button on province view that, together with events and modifiers, could result in colonisation, but I don't have much experience with UI modding. 
For land colonisation, [Wikipedia Article](https://en.wikipedia.org/wiki/Colonization#Classical_period) mentions Roman colonisation briefly and that Qin civilised Asia, but there aren't any references and I can't seem to find a lot of information on this subject. I think it might work for British/Germanic tribes to expand a little bit into the uncolonised areas.

At the very least, I'd like to get rid of the wastelands.
